servers.json

```diff
@@ -1465,5 +1465,14 @@
         "addresses": [
             "pixelcade.net"
         ]
+    },
+    {
+        "id": "alphqhq",
+        "name": "AlphaHQ Network",
+        "addresses": [
+            "alphahq.org",
+            "syrex.dev",
+	    "dev.alphahq.org"
```

Comment:
This address entry is redundant as the system already looks out for sub-domains.